src/vm/jitinterface.cpp

```diff
@@ -10914,27 +10939,7 @@ void CEEJitInfo::reserveUnwindInfo(BOOL isFunclet, BOOL isColdCode, ULONG unwind
 
```

Comment:
@janvorli The first issue while working on #8887 was ``Assert failure(PID 19154 [0x00004ad2], Thread: 19154 [0x4ad2]): iUnwindInfo < GetNumberOfUnwindInfos()``, and the changes in ``Compiler::unwindReserve`` and ``Compiler ::unwindAllocate`` are required to resolve this issue.

You could see that (unlike corresponding implementation for AMD64) the current implementation only stores ``FunctoinLength`` (which is used to implement ``RtlpGetFnctionEndAddress``). It does not store any unwind info, such as which registers are stored, where they are store.

Instead ``RtlVirtualUnwind`` (implemented in #8911) just invokes ``UnwindStackFrame`` which performs unwind based on unwind info embedded in x86 GC info.